packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -21,7 +21,7 @@ AccessibleTabScrollButton.propTypes = {
   direction: PropTypes.string.isRequired,
 };
 
-const findScrollButton = (wrapper, direction) => wrapper.find(`button[aria-label="${direction}"]`);
+const findScrollButton = (wrapper, direction) => wrapper.find(`span[aria-label="${direction}"]`);
```

Comment:
That looks like it's making a11y worse. I stand by https://github.com/mui-org/material-ui/pull/15507#pullrequestreview-231414139. We shouldn't set aria-labels for app authors. If anything we should look at how labels can be provided.

Reply:
This pull request doesn't set aria-labels for app authors. It makes the element disappear from the accessibility tools, they don't need it. How is it worse?

Reply:
a11y is not just for screen readers. non-impaired users make use of these features as well. This looks like it's just hiding an a11y issue instead of solving it.

Reply:
Do you have resources to support this point? The only one I could find is how Antd, Vuetify and Element solve the problem.
I think that we should focus our energy on #6955.